tests/robot-tests/README.md

```diff
@@ -59,137 +58,139 @@ If you intend to run the tests from your local machine, you will also need to cr
+* `RELEASE_COMPLETE_WAIT`: (integer) The amount of time to wait for a release to publish 
+* `WAIT_MEDIUM`: (integer) timeout/sleep variable used throughout the tests 
+* `WAIT_LONG`: (integer) timeout/sleep variable used throughout the tests
+* `FAIL_TEST_SUITES_FAST`: (integer: 0 to 1) Fails test suite after any child test case fails 
+
+
+## **How do I run the tests?**
 
 From the `tests/robot-tests` directory run:
 
 ```
 pipenv run python run_tests.py
 ```
 
-Further instructions on available options
-```
-pipenv run python run_tests.py -h
-```
-
-## Running tests on the pipeline
+This is a python based CLI tool which is responsible for running the UI tests. There are numerous options which are documented below however we typically stick to the following two options:
 
-After changing the tests and ensuring they pass locally, it is then recommended that you run your changes on the pipeline to ensure they also pass there. There is a build pipeline you can use to check any changes you've made will pass on the pipeline: 'UI Tests'. You can run this pipeline using the branch with your changes, and it will produce an artifact you can download with the test report, screenshots, etc.
+- `pipenv run python run_tests.py -f tests/path/to/robot/test -i robot -b chrome -e <environment>`
 
-At the time of writing, the pipeline runs the tests against the Dev environment, and runs all tests. You can edit the pipeline if you wish to do something different.
+- If you have a slow machine or are testing against a slow environment it is advisable to run the tests with the following options: `pipenv run python run_tests.py -f tests/path/to/robot/test -i robot -b chrome -e <environment> --timeout 20 --implicit-wait 20`
 
 
-## Authentication
+Below you will find a set of available CLI options. This is not a complete list of every option that we have available as the intention of this document is to get you started with UI tests & not overwhelm you with the numerous options available.
 
-To run the admin tests, the `run_tests.py` script uses `scripts/get_auth_tokens.py`. The `get_identity_info` function logs in as a user and then returns the relevant local storage and cookies for the authenticated user. This is done so that if an authenticated user is required, a test run only needs to log in once rather than once for each test suite.
+Available CLI options: 
 
-After the user has been logged in by the run_tests.py script, the local storage and cookie data for the authenticated user is saved in the `IDENTITY_LOCAL_STORAGE.txt` and `IDENTITY_COOKIE.txt` files. This is done so that if you're running the tests locally, you don't need to authenticate every time you rerun the tests, as the run_tests.py script will use the data they contain if they exist.
+- `-b` - browser to use (we currently only use Chrome)
 
+- `-i` - interpreter to use. We use robot when testing on our own machines and pabot when testing in CI
```

Comment:
Not so for Nick :)  I reckon best to say use robot when running tests sequentially and use pabot when testing in parallel.